src/lib/consensus/mechanism.ml

```diff
@@ -30,6 +30,10 @@ let env name ~f ~default =
 [%%if
 defined consensus_mechanism]
 
+let network_delay =
```

Comment:
why?

Reply:
I think logic of when to start to start a `bootstrap` should be in embedded in different consensus mechanisms since it's different case-by-case.

Reply:
Sure, that makes sense.